instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/exporters/otlp/OtlpExporterUtil.java

```diff
@@ -39,7 +42,17 @@ static <G, H, E> E applySignalProperties(
     G grpcBuilder = newGrpcBuilder.get();
     H httpBuilder = newHttpBuilder.get();
 
-    boolean isHttpProtobuf = Objects.equals(protocol, OtlpConfigUtil.PROTOCOL_HTTP_PROTOBUF);
+    boolean isHttpProtobuf = !"grpc".equals(protocol);
+
+    if (protocol != null
+        && !OtlpConfigUtil.PROTOCOL_GRPC.equals(protocol)
+        && !OtlpConfigUtil.PROTOCOL_HTTP_PROTOBUF.equals(protocol)) {
+      logger.warn(
+          protocol
+              + " protocol is not managed. "
```

Comment:
I'd rephrase this to something like `Unknown OTLP protocol '{}', using 'http/protobuf' instead.`

Reply:
Done